doc/nrf/known_issues.rst

```diff
@@ -2505,19 +2504,19 @@ NCSDK-15345: TF-M does not support non-secure partitions in external flash
 
   **Workaround:** Do not put non-secure partitions in external flash or keep :kconfig:option:`CONFIG_SPM` enabled in the application.
 
-.. rst-class:: v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0
+.. rst-class:: v2-2-0 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0
```

Comment:
Fixed in 2.2.0

Could we also update the known issue text to say "in library model"?

Reply:
Edited.